pyproject.toml

```diff
@@ -9,7 +9,8 @@ authors = [
     {name = "CFPB", email = "tech@cfpb.gov" }
 ]
 dependencies = [
-    "wagtail>=4.2,<5.2",
```

Comment:
What's the rationale behind increasing the lower bound here?

Reply:
I realize now that wagtail-modeladmin 1.0 requires Wagtail 5.0, so 4.2 wouldn't work here. It'd still be possible to keep 5.0 here if we wanted.

Reply:
Never mind, I realize that wagtail-modeladmin requires Wagtail 5.0+, so there's no way to keep that lower version.